src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/personalrelationships/profiledetails/ContactPersonProfileDetailsSyncIntTest.kt

```diff
@@ -155,9 +154,138 @@ class ContactPersonProfileDetailsSyncIntTest(
     }
   }
 
-  private fun verifyDpsApiCall(prisonerNumber: String = "A1234BC") {
+  @Nested
+  inner class NumberOfChildrenCreatedEvent {
```

Comment:
This was a cut and paste of the domestic status tests but tweaked for number of children